p2p/node.go

```diff
@@ -181,11 +197,9 @@ func (n *QriNode) StartOnlineServices(bootstrapped func(string)) error {
 	// need a call here to ensure boostrapped is called at least once
 	// TODO - this is an "original node" problem probably solved by being able
 	// to start a node with *no* qri peers specified.
-	defer bootstrapped("")
 
 	go func() {
-		pInfo := <-bsPeers
-		bootstrapped(pInfo.ID.Pretty())
+		_ = <-bsPeers
```

Comment:
this can be refactored to just `<-bsPeers` without the assignment. It'd be great to add a comment that says: `block until we have at least one successful bootstrap connection`